accepted/future-releases/static-extension-methods/feature-specification.md

```diff
@@ -315,12 +315,20 @@ or if you imported the extension with a prefix to avoid name collision:
 prefix.MyList<String>(object).quickSort();
 ```
 
-The syntax looks like a constructor invocation, but it does not create a new object.
+The syntax looks like a constructor invocation, but it does not create a new object. Instead it evaluates `object` to a value, then calls the `quickSort` extension method of `MyList` on that object.
```

Comment:
Consider making it explicit that `MyList(o).foo` is an error if `foo` is not an extension member of `MyList`, even if `foo` *is* an instance member on `o`?

Reply:
Will do.